docs/sample_config.yaml

```diff
@@ -784,22 +784,33 @@ caches:
   #
   #cache_entry_ttl: 30m
 
-  # This flag enables cache autotuning, and is further specified by the sub-options `max_cache_memory_usage`,
-  # `target_cache_memory_usage`, `min_cache_ttl`. These flags work in conjunction with each other to maintain
-  # a balance between cache memory usage and cache entry availability. You must be using jemalloc to utilize
-  # this option, and all three of the options must be specified for this feature to work.
+  # This flag enables cache autotuning, and is further specified by the
+  # sub-options `max_cache_memory_usage`, `target_cache_memory_usage`,
+  # `min_cache_ttl`. These flags work in conjunction with each other to
+  # maintain a balance between cache memory usage and cache entry
+  # availability. You must be using jemalloc to utilize this option, and
+  # all three of the options must be specified for this feature to work.
+  #
   #cache_autotuning:
-    # This flag sets a ceiling on much memory the cache can use before caches begin to be continuously evicted.
-    # They will continue to be evicted until the memory usage drops below the `target_memory_usage`, set in
-    # the flag below, or until the `min_cache_ttl` is hit.
+    # This flag sets a ceiling on much memory the cache can use before
+    # caches begin to be continuously evicted. They will continue to be
+    # evicted until the memory usage drops below the
+    # `target_memory_usage`, set in the flag below, or until the
+    # `min_cache_ttl` is hit.
+    #
     #max_cache_memory_usage: 1024M
```

Comment:
please could you document, here and in the config manual, what the valid suffixes are, and what they mean? (is M 1000000 or 1048576 ?)